File: sk000801/LEVEL1/푸드 파이트 대회.java
class Solution {
    public String solution(int[] food) {
        String answer = "";
        for(int i = 1; i < food.length; i++) {
            if(food[i] > 1) {
                for(int j = 1; j <= food[i]/2; j++) {
                    answer += i;
                }
            }
        }
        String reverse = "";
        for(int i = 0; i < answer.length(); i++) {
            reverse += answer.charAt(answer.length()-1-i);
        }
        String realAnswer = answer+ "0" + reverse;
        return realAnswer;
    }
}

File: kyw0716/level 1/푸드 파이트 대회.js
function solution(food) {
  let answer = "";
  let foodString = [];

  for (let i = 1; i < food.length; i++) {
    for (let j = 0; j < Math.floor(food[i] / 2); j++) {
      foodString.push(i);
    }
  }

  answer += foodString.join("");
  answer += "0";
  answer += foodString.reverse().join("");

  return answer;
}

//https://school.programmers.co.kr/learn/courses/30/lessons/134240
